elements/rh-breadcrumb/rh-breadcrumb-lightdom.css

```diff
@@ -0,0 +1,121 @@
+rh-breadcrumb {
```

Comment:
these can go on `:host` in shadow dom css

Reply:
Fixed in [864f756](https://github.com/RedHat-UX/red-hat-design-system/pull/1535/commits/864f75686b86bc33dde1609ca47bba3bd7e183df).